packages/ingest/pdr.js

```diff
@@ -1,9 +1,9 @@
 'use strict';
 
-const aws = require('@cumulus/common/aws');
 const fs = require('fs-extra');
 const { ftpMixin } = require('./ftp');
 const get = require('lodash.get');
+const getPdr = require('@cumulus/api/endpoints/pdrs').get;
```

Comment:
I think you could do something here like 
`const { get: getPdr } = require('@cumulus/api/endpoints/pdrs')`

Reply:
👍 I'm a fan